docs/concepts/tokens/fungible-tokens/index.md

```diff
@@ -1,93 +1,58 @@
 
+Some notable features and their status:
 
-## Trust Line Settings
+| MPT Feature | Amendment / Standard |
+|-------------|----------------------|
+| Issuing and direct payments | {% amendment-disclaimer name="MPTokensV1" compact="true" /%} |
+| Escrow | {% amendment-disclaimer name="TokenEscrow" compact="true" /%} (implements escrow for both trust line tokens and MPTs) |
+| Mutable token properties | In development: [XLS-94](https://github.com/XRPLF/XRPL-Standards/tree/master/XLS-0094-dynamic-MPT) |
+| Confidentiality | Proposed: [XLS-??](https://github.com/XRPLF/XRPL-Standards/discussions/372) |
+| Decentralized exchange (DEX), trading, cross-currency payments, and AMM compatibility | Expected future work |
 
-In addition to the shared balance, each account has its own settings on the trust line, which consist of the following:
+## Which Fungible Token Type to Use
 
-- The **Limit**, a number from 0 to the [maximum token amount](../../../references/protocol/data-types/currency-formats.md). Payments and other accounts' actions cannot cause the trust line's balance (from this account's perspective) to go over the limit. The default is `0`.
-- **Authorized**: A true/false value used with [Authorized Trust Lines](authorized-trust-lines.md) to allow the other side to hold tokens this account issues. The default is `false`. Once set to `true`, this cannot be changed back.
-- **No Ripple**: A true/false value to control whether tokens can [ripple](rippling.md) through this trust line. The default depends on the account's "Default Ripple" setting; for new accounts, "Default Ripple" is off which means that `true` is the default for No Ripple. Usually, issuers should allow rippling and non-issuers should disable rippling unless they are using trust lines for community credit.
-- **Freeze**: A true/false value indicating whether an [individual freeze](freezes.md#individual-freeze) is in effect on this trust line. The default is `false`.
-- **DeepFreeze**: A true/false value indicating whether a [deep freeze](deep-freeze.md) is in effect on this trust line. The default is `false`.
-- **Quality In** and **Quality Out** settings, which allow the account to value tokens issued by the other account on this trust line at less (or more) than face value. For example, if a stablecoin issuer charges a 3% fee for withdrawing tokens for the equivalent off-ledger assets, you could use these settings to value those tokens at 97% of face value. The default, `0`, represents face value.
+If you are creating a new token on the XRP Ledger, it may be confusing that there are two standards to choose from with different properties and strengths. If you aren't sure which one to use, the following summary should help you decide:
 
+- If you need **compatibility with the DEX** or your use case is **community credit**, choose trust line tokens.
```

Comment:
Can we define what community credit is supposed to mean? Also maybe de-emphasize the dex compatibility since the reason for this is that MPTs just don't have it yet--don't want to create the idea that MPTs just aren't good for dex in general.

Reply:
As for your other question, "community credit" is not easy to look up, so I agree that it's an important gap. Unfortunately, it's also a pretty involved topic, somewhat fundamental with the design of the original "Ripple Network" (the circa 2009, pre-XRP one, even), so it's not really something that we can just slip into these docs.

#1353 is the existing ticket for better documenting community credit, so I'd like to continue to leave that one as a separate task.